docs/cloud/managing-airbyte-cloud.md

```diff
@@ -274,9 +276,75 @@ Airbyte recommends that you reset streams. A reset will delete data in the desti
 To refresh the source schema:
 1. Click **Refresh source schema** to fetch the schema of your data source.
 
-2. If there are changes to the schema, the **Refreshed source schema** dialog displays them.
+2. If the schema has changed, the **Refreshed source schema** dialog displays them.
+
+## Manage schema changes
+
+Airbyte detects changes in your source schema and allows you to review the changes.
+
+:::note 
+
+* Schema changes are flagged in your connection, but they will not affect your destination. 
```

Comment:
All three points don't seem to have the same level of importance. I'd leave the point about schema changes not being propagated to the destination as a note, and make the other two points simple statements.

I'd write the start of the section as follows:

Once every 24 hours, Airbyte checks for changes in your source schema and allows you to review the changes and fix breaking changes. 

:::note 

Schema changes are flagged in your connection but are not propagated to your destination. 
:::

I know I said we could put the line about breaking changes at the top of the section, but with the new content, I think we can move it to the top of the section about breaking changes. Let me know what you think.